Letterbook.Core/Models/Actor.cs

```diff
@@ -9,9 +9,16 @@
 /// </summary>
 public class Actor
 {
+    private Actor()
+    {
```

Comment:
I'm not sure I understand mixing where these types are initialized (constructor vs property initializer). Can you help me understand this?

Reply:
This is an efcore thing. It does some special magic to get access to the private default constructor, uses that to create the object, and then uses the accessors to set values on everything.

So, this does some null forgiveness for the non-nullable fields, to satisfy the compiler. But if we're creating these objects in user code, we'll still get compiler errors about leaving them unitialized.